data/coin-dd.json

```diff
@@ -2,7 +2,7 @@
     "distance_type" : "None",
     "termination_crit" : "genmax",
     "termination_args" : 30,
-    "pop_num" : 40,
+    "pop_num" : 50,
```

Comment:
Please, leave this file unchanged, and create a new one, named e.g. biokdd-dd.json